lib/ignored-paths.js

```diff
@@ -16,12 +16,12 @@ const fs = require("fs"),
 
 const debug = require("debug")("eslint:ignored-paths");
 
-
 //------------------------------------------------------------------------------
 // Constants
 //------------------------------------------------------------------------------
 
 const ESLINT_IGNORE_FILENAME = ".eslintignore";
+const ESLINT_PACKAGE_JSON = "package.json";
```

Comment:
Nitpick: Can you rename this constant to something like `PACKAGE_JSON_FILENAME`? The filename `package.json` isn't eslint-specific.

(Actually, it might not be necessary to use a constant for this at all, since it's a Node convention that probably isn't going to change.)